tests/wicked/basic/t01_basic.pm

```diff
@@ -63,4 +63,8 @@ sub run {
     validate_script_output("wicked show dev $iface",     sub { m/\[dhcp\]/g; });
 }
 
+sub test_flags {
+    return {always_rollback => 1, wicked_need_sync => 1};
```

Comment:
We do not need `wicked_need_sync` as long as we do not have a SUT and REF job.
Same on the other files.

```suggestion
    return {always_rollback => 1};
```